packages/snaps-sdk/src/types/methods/manage-state.ts

```diff
@@ -0,0 +1,30 @@
+import type { Json } from '@metamask/utils';
+
+import type { EnumToUnion } from '../../internals';
```

Comment:
What's the goal of internals? Types that aren't directly exported but used to define exported types?

Reply:
Right now just types, but in the future it may be functions etc. But yeah, it's stuff that doesn't need to be exported in the public API, but are useful for creating the public API.